functional-test/src/test/resources/projects-import/webhooks-test/rundeck-importProjectWithWebhook/webhooks.yaml

```diff
@@ -0,0 +1,4 @@
+webhooks:
```

Comment:
if this is only used in PRO, it should go in PRO repo

Reply:
it stays in oss